### DISCUSSION TITLE
Disable Automatic Check for Update on DWSIM Startup

### DISCUSSION THREAD

**Discussion by kookma** [DanWBR/dwsim#335] [General]
When we use DWSIM in a class, we like to have the same version during the class.
DWSIM on start shows automatically a message window when a new version is available, this may be annoying if you want to stick the same version or students may click and download the new version which is problematic as instructor goes with an older version.

It would be great to have an option to enable/disable check for updates on DWSIM startup.

**Comment by DanWBR** [comment]
@kookma added a checkbox to disable update checks to the general settings panel in v8.0.2.
